requirements.txt

```diff
@@ -1,7 +1,8 @@
 six
-wxPython>=4.0,<4.2.0 ; platform_system=="Linux" # See https://github.com/wxWidgets/Phoenix/issues/2225
+# See https://github.com/wxWidgets/Phoenix/issues/2225
+wxPython@https://wxpython.org/Phoenix/snapshot-builds/wxPython-4.2.1a1.dev5539+906adf71.tar.gz ; platform_system=="Linux"
```

Comment:
Doesn't this mean that even on a system with a perfectly-working dist-packages-based wxPython, that pip will force the user to install this snapshot build? I think that we should be able to go back to what I had before for Linux and rely on wx being optional in the regular setup.py install for the case where people will use dist packages right?

BTW, I assume what you care about is BSD...none of these targets match it right?

Reply:
This file is not read at all unless either explicitly passed into `pip -r` or `pip` itself can't get out of its own way resolving dependencies, which with `install_required` is basically never. This file is a non-factor when not using `pip` as it's specific to that tool.

Reply:
I know it's not read in your use-case, but it's used when people are trying to get a development environment running, and I use it for the bundled builds. If there's not a need to make this change, I'd prefer to go back to what I had.

Reply:
It's not read even in that development environment case, as illustrated with the earlier busted tests. I had to stick the specification into `test-requirements.txt` to avoid installing the broken wxPython 4.2.0.